.github/workflows/package-manager-demo.yml

```diff
@@ -11,42 +11,51 @@ permissions:
 
 jobs:
   build:
-    env:
-      PACKAGEMANAGER_TOKEN_SOLO: ${{ secrets.PACKAGEMANAGER_TOKEN_SOLO }}
-      PACKAGEMANAGER_TOKEN_CLUSTER: ${{ secrets.PACKAGEMANAGER_TOKEN_CLUSTER }}
-      PACKAGEMANAGER_TOKEN_S3: ${{ secrets.PACKAGEMANAGER_TOKEN_S3 }}
     strategy:
       matrix:
         r-version: ['3.6.3', '4.1.3']
         os: [ {os: "ubuntu-20.04", distro: "focal"}, {os: "ubuntu-18.04", distro: "bionic"} ]
         package-manager: [
-          {token: "$PACKAGEMANAGER_TOKEN_SOLO", endpoint: "https://solo.rstudiopm.com"},
-          {token: "$PACKAGEMANAGER_TOKEN_S3", endpoint: "https://s3.rstudiopm.com"},
-          {token: "$PACKAGEMANAGER_TOKEN_CLUSTER", endpoint: "https://cluster.rstudiopm.com"},
+          {endpoint: "https://solo.rstudiopm.com", token: "PACKAGEMANAGER_TOKEN_SOLO"},
+          {endpoint: "https://s3.rstudiopm.com", token: "PACKAGEMANAGER_TOKEN_S3"},
+          {endpoint: "https://cluster.rstudiopm.com", token: "PACKAGEMANAGER_TOKEN_CLUSTER"},
         ]
     runs-on: ${{ matrix.os.os }}
     steps:
       - uses: actions/checkout@v3
       - name: Set up R ${{ matrix.r-version }}
-        uses: r-lib/actions/setup-r@f57f1301a053485946083d7a45022b278929a78a
+        uses: r-lib/actions/setup-r@v2
```

Comment:
I changed this to the v2 tag just because it's what most users will be doing, following the https://github.com/r-lib/actions/ docs. If there was a specific reason for pinning to this commit, I'll revert this.